test/IIS.FunctionalTests/DeployerSelector.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    public static class DeployerSelector
+    {
+        public static ServerType ServerType => ServerType.IIS;
+        public static bool IsBackwardsCompatiblityTest => false;
```

Comment:
Why not in RequiresNewShimAttribute?

Reply:
We already used DeployerSelector for specifying whether a test was IIS or IISExpress (for statics). Seemed better to use this instead.